step.rb

```diff
@@ -39,7 +49,27 @@
 puts "Starting step with path: #{project_path}, scheme: #{scheme}, trace version: #{lib_version}"
 puts
 
-FileUtils.mv(tmpf.path, "#{File.dirname(project_path)}/#{tmpf.original_filename}")
+fileLocation = "#{File.dirname(project_path)}/#{tmpf.original_filename}"
+FileUtils.mv(tmpf.path, fileLocation)
+
+# Unzip file
+
+puts "Unzipping Trace SDK"
+
+Zip::File.open(fileLocation) do |zip_file|
+    zip_file.each do |f|
+        if f.name == "libTrace.a"
+            fpath = File.join("#{File.dirname(project_path)}", f.name)
+            zip_file.extract(f, fpath) unless File.exist?(fpath)
+            
+            puts "Unzipping: #{fpath}"
+        else 
+            puts "Skipping file: #{f.name}"
```

Comment:
Question: If there are subdirectories multiple files can have the same name, would it worth to add the relative path here or there would be no benefit?

Reply:
it's a flat structure, I don't see this changing anytime